app/models/pdc_serialization/datacite.rb

```diff
@@ -83,6 +83,7 @@ def self.new_from_work(work)
 
     ##
     # Creates a PDCSerialization::Datacite object from a PDCMetadata::Resource
+    # rubocop: disable Metrics/MethodLength
```

Comment:
I think it makes sense to keep this method as a single one call to `::Datacite::Mapping::Resource.new` so I disabled the warning.